### DISCUSSION TITLE
Consiglio su dispositivo

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2237] [Hardware & Firmware]
Ciao a tutti, vorrei un consiglio su quale dispositivo acquistare da collegare al mio HA.
Dispositivo 1 : deve leggere lo stato di tre contatti (scheda antifurto) e non posso usare i dispositivi che usi il 220V in quanto i contatti non li supportano.
Dispositivo 2 : deve dare impulsi (chiudere un contatto). Lo collego a due telecomandi apricancello.
Grazie a tutti in anticipo!

**Comment by sky7176** [comment]
Ciao,
Dispositivo 1: devi fare un antifurto oppure legge lo stato di un antifurto esistente?
Dispositivo 2: 
        Soluzione 1:Io ho risolto con un esp01 con uscita rele, l'uscita, che è un contatto pulito, l'ho saldata in parallelo al pulsante del 
                           telecomando, esp01 con tasmota a bordo e comando il tutto con MQTT, l'unica cosa è che quelli che trovi in giro devono
                           essere alimentati a 5 Vcc
        Soluzione 2: Prendi un Sonoff ch4pro R3 lo riprogramma con un firmware Tasmota,che è alimentato a 230 Volt, ha 4 uscite rele a
                            contatti puliti,non hanno tensione in uscita, e ci comandi i due telecomandi

Se hai bisogno sono a disposizione, io un giochino del genere l'ho fatto per il carraio di casa.

Giuseppe

**Reply by Matty1184** [reply]
> Dispositivo 1: devi fare un antifurto oppure legge lo stato di un antifurto esistente?

Devo attaccarlo a un antifurto esistente che ha 3 uscite già programmate con contatti puliti. 
pensavo a Shelly Plus i4 DC alimentato a 24DC

> Dispositivo 2:
> Soluzione 1:Io ho risolto con un esp01 con uscita rele, l'uscita, che è un contatto pulito, l'ho saldata in parallelo al pulsante del
> telecomando, esp01 con tasmota a bordo e comando il tutto con MQTT, l'unica cosa è che quelli che trovi in giro devono
> essere alimentati a 5 Vcc
> Soluzione 2: Prendi un Sonoff ch4pro R3 lo riprogramma con un firmware Tasmota,che è alimentato a 230 Volt, ha 4 uscite rele a
> contatti puliti,non hanno tensione in uscita, e ci comandi i due telecomandi

sulla prima proposta non mi voglio cimentare al momento volevo se possibile qualcosa di già pronto. sulla seconda la vedo già alla mia portata. domansa.. .perchè dovrei riprogrammarlo con il tasmota? con il suo fw non funzionarebbe?
ho verificato se esisteva uno shelly simile ma vedo che tutti utilizzano i 220v come contatto (non pulito)

**Reply by sky7176** [reply]
Dispositivo 1: dispositivi che accettino un ingresso non ne conosco di quelli commerciali.
Dispositivo 2: Sostituire il firmware originale con il Tasmota non è obbligatorio, io ho tutto con Tasmota, è semplicemente una questione di gusti personali.

Giuseppe
